src/libsaml.ts

```diff
@@ -108,6 +109,28 @@ export interface LibSamlInterface {
 }
 
 const libSaml = () => {
+  const validator = new Validator();
+  function setSchemaDir() {
+    let schemaDir;
+    try {
+      schemaDir = path.resolve(__dirname, '../schemas');
+      fs.accessSync(schemaDir, fs.constants.F_OK);
+    } catch (err) {
+      // for built-from git folder layout
+      try {
+        schemaDir = path.resolve(__dirname, '../../schemas');
```

Comment:
Isn't `../schemas` enough?

Reply:
Yes, most likely. When I tried using directly from git (rather than the npm package) I ended up with the schema folder one level up. May very well have been my mistake - I had to disable the "preinstall" npm script because I'm building on Windows and the shell script assumes bash.